supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala

```diff
@@ -119,7 +136,13 @@ sealed trait ConfigAsyncJobExecutionActor extends SharedFileSystemAsyncJobExecut
   * @param standardParams Params for running a shared file system job.
   */
 class BackgroundConfigAsyncJobExecutionActor(override val standardParams: StandardAsyncExecutionActorParams)
-  extends ConfigAsyncJobExecutionActor with BackgroundAsyncJobExecutionActor
+  extends ConfigAsyncJobExecutionActor with BackgroundAsyncJobExecutionActor {
+
+  override def killArgs(job: StandardAsyncJob): SharedFileSystemCommand = {
+    if (isDockerRun) jobScriptArgs(job, "kill", KillDockerTask, Map(DockerCidInput -> dockerCidInputValue))
```

Comment:
ToL: This will need to be revisited when we finally get UGE docker [support](https://blogs.univa.com/2016/05/new-version-of-univa-grid-engine-now-supports-docker-containers/)